opencensus/trace/ext/cloud_clientlibs/trace.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import grpc
+
+from google.cloud import _helpers
+
+from opencensus.trace import execution_context
+from opencensus.trace.ext.grpc.client_interceptor import (
+    OpenCensusClientInterceptor)
+
+from opencensus.trace.ext.requests.trace import (
+    trace_integration as trace_requests)
+
+log = logging.getLogger(__name__)
+
+MODULE_NAME = 'cloud_clientlibs'
```

Comment:
Should probably call this 'google_cloud_clientlibs' or something, so that it doesn't give special preference to google over other client libs.

Reply:
Renamed.